bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/monopole_SpinHalf_DrellYan/cards/SpinHalf_DrellYan_1000/monopole_SpinHalf_DrellYan_M1000_run_card.dat

```diff
@@ -0,0 +1,129 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update to_full                                                *
+#*********************************************************************
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+#*********************************************************************
+  100000	= nevents ! Number of unweighted events requested 
+  0	= iseed ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                3=photon from electron, 4=photon from muon          *
+#*********************************************************************
+  1	= lpp1 ! beam 1 type 
+  1	= lpp2 ! beam 2 type
+  6500.0	= ebeam1 ! beam 1 total energy in GeV
+  6500.0	= ebeam2 ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf	= pdlabel ! PDF set                                     
+  324900	= lhaid ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
Instead of using the default `$DEFAULT_PDF_SETS` syntax you specify the 324900 pdf set (`NNPDF31_nlo_as_0118_luxqed`). Is it done on purpose?

Reply:
Yes, it is also done on purpose, we are considering this specific pdf only for the case where we have photon fusion. The monopole has a particularity that needs to be considered for this production mode using qed corrections when we use photon fusion. This has already been pointed out in some papers that we use as reference.